examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+"""
+==================================================================
+Demonstrate morphing subject source volume to fsaverage using dipy
+==================================================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose dipy (http://nipy.org/dipy/) will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example lcmv beamformer
+pipeline (examples/inverse/plot_lcmv_beamformer_volume.py).
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain). Afterwards the
+transformation will be applied to the beamformer result. Affine transformations
+are computed based on the mutual information and nonlinear transformations
+will be performed as Symmetric Diffeomorphic Registration using the
+cross-correlation metric [1]_
+
+.. warning:: Please do not copy the patterns presented here for your own
+             analysis, this is example is purely illustrative.
```

Comment:
Is this warning relevant? What here is bad practice?

Reply:
nope just was in the example I copied and I just assumed it to be some default reassuring thing since it's an example... I'll remove